component/grid/grid.css

```diff
@@ -0,0 +1,547 @@
+@import url('https://fonts.googleapis.com/css2?family=Redressed&family=Roboto:wght@300&family=Satisfy&family=Shizuru&display=swap');
+
+body{
+font-family: 'Roboto', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color: #111827;
+}
+
+* {
+
+--primary-color: #ecfeff ;
+--secondary-color: #FA0874;
+--black: #111827;
+}
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-auto-flow: column;
+    grid-template-columns: 200px 1fr; 
+ 
+    grid-template-areas: 
+    "nav nav "
+    "pitch pitch"
+   "aside main"
+   "aside main"
+   "aside main"
+   "footer footer"
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+    
+
+}
+
+nav h2{
+    position: relative;
+    top: 35px;
+    font-size: 2rem;
+    line-height: 2.5rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto; 
+    display: flex;
+  gap: 15px;
+  position: relative;
+  bottom: 35px;
+  padding-inline-end: 0.7rem;
+ 
+}
+
+li{
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+   
+    grid-area: main;
+    font-size: 1.2rem;
+    padding-left: 7rem;
+    padding-right: 7rem;
+    
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+aside {
+
+    background: #111827;
+}
+
+aside label{
+    color: var(--black);
+}
+
+aside #hamburger{
+grid-area: ham;
+display: none;
+
+}
+
+h3{
+    font-size: 1.5rem;
+    /* line-height: 2rem; */
+    letter-spacing: 0.1rem;
+ }
+
+ .impact{
+     font-style: italic;
+     color: var(--secondary-color);
+     
+ }
+
+ 
+ 
+
+.component-text{
+    grid-area: pitch;
+    font-size: 2rem;
+    margin: auto;
+    letter-spacing: 0.1rem;
+}
+
+
+hr{
+    border-top: 1px solid var(--primary-color); 
+  width: 50rem;
+}
+
+.highlight{
+    color: var(--secondary-color);
+}
+
+iframe{
+    padding-top: 1rem;
+}
+
+
+
+
+.medium{
+    width: 4.5rem;
+    height: 4.5rem;
+    margin-left: 2rem; 
+}
+
+.large{
+    width: 5.5rem;
+    height: 5.5rem;
+    margin-left: 2rem; 
+}
+
+.bold{
+    font-weight: 600;
+    font-size: 1.5srem;
+    padding-top: 3.5rem;
+}
+
+
+
+
+.grid-2 {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+   
+}
+
+.grid-items {
+    padding: 1rem;
+    border: 1px solid var(--secondary-color);
+    margin: 0.5rem;
+    height: 3rem;
+    border-radius: 0.5rem;
+}
+
+.grid-3 {
+    display: grid;
+    grid-template-columns: 1fr;
+    

```

Comment:
i think here you gave 1 fr that's why they went to next lines. as @zabihshaik  mentioned for column grids they would have probably came under one row.

Reply:
oh thanks